src/ServicePulse.Host/vue/src/components/PaginationStrip.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { expect, it, render, screen, describe } from "@component-test-utils";
+import paginationStrip from "./PaginationStrip.vue";
+
+describe("Previous page behavior", () => {
+  it("Disables navigating to previous page while on first page", async () => {
+    render(paginationStrip, {
+      props: {
+        modelValue: 1,
+        itemsPerPage: 0,
+        totalCount: 100,
+        pageBuffer: 0,
+      },
+    });
+
+    expect(screen.queryByLabelText("Previous")).toBeDisabled();
+  });
+
+  it("Enables navigating to 'Previous' page while not on first page", async () => {
+    render(paginationStrip, {
+      props: {
+        modelValue: 2,
+        itemsPerPage: 0,
+        totalCount: 100,
+        pageBuffer: 0,
+      },
+    });
+
+    expect(screen.queryByLabelText("Previous")).not.toBeDisabled();
+  });
+});
+
+describe("Next page behavior", () => {
+  it.todo("Disables navigating to 'Next' page while in the last page");
+  it.todo("Enables navigating to 'Next' page while NOT in the last page");
+});
+
+describe("Page buffer behavior", () => {
+  it.todo("Example 1");
+  it.todo("Example 2");
+});
+
+describe("Total count and items per page behavior", () => {
+  it.todo("Example 1");
+  it.todo("Example 2");
+});
```

Comment:
@mikeminutillo could you suggest scenarios for testing the page bugger behavior? I can implement the tests.

Reply:
The page buffer indicates how many pages to the left and right of the current page before showing an elipsis `...`.

- If the current page number is less than the page buffer there should be no left ellipsis
- If the current page number is greater than the page buffer there should be a left ellipsis
- If the current page is equal to the page buffer then should be no left ellipsis
Same for the right ellipsis, although that should be "current page is within page buffer of the last page.